ch02/groovybaseball/build.gradle

```diff
@@ -57,7 +57,7 @@ task wrapper(type:Wrapper) {
 }
 
 dependencies {
-    compile 'org.codehaus.groovy:groovy-all:2.4.14'
+    compile 'org.codehaus.groovy:groovy-all:2.4.12'

```

Comment:
There seems to be incompatibility between Jetty annotation parser and groovy:2.4.14. My quick fix is to use older version of groovy.

Reply:
Yes, this fixed the problem. Thanks! Hopefully you can do a longer term fix later.